pages/linux/cpuid.md

```diff
@@ -0,0 +1,15 @@
+# cpuid
+
+> Display detailed information about all CPUs.
+
+- Display information for all CPUs:
+
+`cpuid`
+
+- Display information only for the first CPU:
```

Comment:
Is there an option to display info only for, say, the third CPU too?

Reply:
No, I haven't found.
I've updated the confusing description.
`Display information only for the current CPU`
Not `the first CPU`.

Reply:
What's meant by `current` cpu?

Reply:
`cpuid -1` captures the current cpu on the multiprocessor system,